app/src/main/java/org/systers/mentorship/view/fragments/MembersFragment.kt

```diff
@@ -127,6 +131,23 @@ class MembersFragment : BaseFragment() {
                                 isRecyclerView = true
                             }
                         }
+                        else
+                        {
+                            if(filterMap.containsKey("location")&&filterMap["location"] != "")
+                            {
+                                val User_list = membersViewModel.userList
```

Comment:
This as well

Reply:
User_list variable is kind of necessary, cause without this idk why but the result always comes "false". But will try again.

Reply:
> User_list variable is kind of necessary, cause without this idk why but the result always comes "false". But will try again.
@epicadk 
There was an issue I faced earlier, userlist size is not consistent, it changes everytime I change the location name. Sometimes the arraylist size is 20 only and sometimes it shows 140.